desktopexporter/internal/store/store_test.go

```diff
@@ -231,3 +231,230 @@ func TestTraceSummaryOrdering(t *testing.T) {
 	assert.NotNil(t, summaries[1].RootSpan, "trace1 should have root span")
 	assert.NotNil(t, summaries[0].RootSpan, "trace3 should have root span")
 }
+
+func TestAttributeTypes(t *testing.T) {
+	ctx := context.Background()
+	store := NewStore(ctx, "")
+	defer store.Close()
+
+	// Create a test span with all supported attribute types
+	baseTime := time.Now()
+	spans := []telemetry.SpanData{
+		{
+			TraceID:      "test-trace-1",
+			SpanID:       "test-span-1",
+			ParentSpanID: "",
+			Name:         "test-span",
+			Kind:         "SPAN_KIND_SERVER",
+			StartTime:    baseTime,
+			EndTime:      baseTime.Add(time.Second),
+			Attributes: map[string]any{
+				"string1":      "Koala",
+				"string2":      "Bear",
+				"bigint1":   int64(42),
+				"double":   float64(3.14),
+				"boolean":  true,
+				"str_list": []string{"one", "two", "three"},
+				"int_list": []int64{1, 2, 3},
+				"float_list": []float64{1.1, 2.2, 3.3},
+				"bool_list": []bool{true, false, true},
+			},
+			Resource: &telemetry.ResourceData{
+				Attributes: map[string]any{},
+				DroppedAttributesCount: 0,
+			},
+			Scope: &telemetry.ScopeData{
+				Name:    "test-scope",
+				Version: "v1.0.0",
+				Attributes: map[string]any{},
+				DroppedAttributesCount: 0,
+			},
```

Comment:
I know they use the same code path, but testing resource and scope attributes here would be nice

Reply:
I will add this in my next db pr.